### ISSUE TITLE
Deferred Block Calls (akin to WinNT DPCs)

### ISSUE THREAD

**Issue by netbsduser** [NetaScale/Valutron#5]
For Valutron to handle interrupts and suchlike, a mechanism akin to Windows' DPC is necessary, i.e. to allow the enqueueing of some work to run on a particular CPU when appropriate. We can call them 'Deferred Block Calls' as the plan is to implement them in terms of Valutron blocks.
This could probably just be a low-priority thread waiting on some semaphore which is signalled whenever a DBC is enqueued - the DBC then runs un-preemptably (but other interrupts occurring during interruption may enqueue further DBCs.)
